backend/molgenis-emx2-rdf/src/test/java/org/molgenis/emx2/rdf/ColumnTypeRdfMapperTest.java

```diff
@@ -282,40 +283,38 @@ void validateAllColumnTypesCovered() {
   void validateValueTypes() {
     Row row = allColumnTypes.getTable(TEST_TABLE).retrieveRows().get(0);
 
-    assertAll(
-        // SIMPLE
-        () -> assertTrue(retrieveFirstValue(ColumnType.BOOL.name()).isLiteral()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.UUID.name()).isIRI()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.FILE.name()).isIRI()),
-        // STRING
-        () -> assertTrue(retrieveFirstValue(ColumnType.STRING.name()).isLiteral()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.TEXT.name()).isLiteral()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.JSON.name()).isLiteral()),
-
-        // NUMERIC
-        () -> assertTrue(retrieveFirstValue(ColumnType.INT.name()).isLiteral()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.LONG.name()).isLiteral()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.DECIMAL.name()).isLiteral()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.DATE.name()).isLiteral()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.DATETIME.name()).isLiteral()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.PERIOD.name()).isLiteral()),
-
-        // RELATIONSHIP
-        () -> assertTrue(retrieveFirstValue(ColumnType.REF.name()).isIRI()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.REFBACK.name()).isIRI()),
-
-        // LAYOUT and other constants
-        // ColumnType.HEADING.name() -> no Value should be present to validate on
-
-        // format flavors that extend a baseType
-        () -> assertTrue(retrieveFirstValue(ColumnType.AUTO_ID.name()).isLiteral()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.ONTOLOGY.name()).isIRI()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.EMAIL.name()).isIRI()),
-        () -> assertTrue(retrieveFirstValue(ColumnType.HYPERLINK.name()).isIRI()),
-
-        // Composite keys
-        () -> assertTrue(retrieveFirstValue(COLUMN_COMPOSITE_REF).isIRI()),
-        () -> assertTrue(retrieveFirstValue(COLUMN_COMPOSITE_REFBACK).isIRI()));
+    assertTrue(retrieveFirstValue(ColumnType.BOOL.name()).isLiteral());
```

Comment:
made it easier to find what assert failed.

Reply:
Seems this got reverted (which I'm happy with as otherwise you don't know if multiple fail at the same time). While you can check which one failed by the line number, I suggest adding custom error messages instead:
```
  @Test
  void example() {
    Boolean test = FALSE;
    assertAll(
        () -> assertTrue(test),
        // comment otherwise auto-wrap makes single line due to short example
        () -> assertTrue(test, "custom fail message"));
  }
```
Snippet of error log:
```
org.opentest4j.MultipleFailuresError: Multiple Failures (2 failures)
	org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
	org.opentest4j.AssertionFailedError: custom fail message ==> expected: <true> but was: <false>
```

Reply:
PR ready with fix: https://github.com/molgenis/molgenis-emx2/pull/4744